.github/workflows/pytest.yml

```diff
@@ -241,6 +241,9 @@ jobs:
           echo "PERL_MB_OPT=--install_base \"$HOME/perl5\"" >> $GITHUB_ENV
           echo "PERL_MM_OPT=INSTALL_BASE=$HOME/perl5" >> $GITHUB_ENV
           echo "$HOME/perl5/bin" >> $GITHUB_PATH
+      - name: Install Fortran Language Server
+        shell: bash
+        run: pip install fortls
```

Comment:
Since fortls is in python, we could install it as part of serena (like pyright). Maybe as an extra, although it would require users to modify their uvx command.

@opcode81 wdyt? It would increase package size, but maybe not unacceptably so

Reply:
made changes but for some reason tests on mac-os fail because of a unrelated C#  test, i will mark this resolved.